rust/actyx/ax/src/cmd/settings/local/get.rs

```diff
@@ -0,0 +1,61 @@
+use crate::cmd::{determine_ax_default_data_dir, settings::get::GetSettingsCommand, AxCliCommand};
+use ax_core::{
+    node::{self, lock_working_dir},
+    util::formats::{ActyxOSCode, ActyxOSError, ActyxOSResult},
+};
+use futures::{stream, Stream};
+use std::path::PathBuf;
+
+#[derive(clap::Parser, Debug, Clone)]
+pub struct SettingsLocalGetOpts {
+    #[command(flatten)]
+    actual_opts: GetSettingsCommand,
+    /// Path where to store all the data of the Actyx node.
+    #[arg(
+        long,
+        env = "ACTYX_PATH",
+        long_help = "Path where to store all the data of the Actyx node. \
+          Defaults to creating <current working dir>/actyx-data"
+    )]
+    working_dir: Option<PathBuf>,
+}
+
+pub struct SettingsLocalGet();
```

Comment:
Why the `()`?

Reply:
I copied this the struct from the old `setting`. The `()` is now removed.